Tests/TemplatesTests/JsonTests.swift

```diff
@@ -9,7 +9,7 @@ import XCTest
 
 final class JsonTests: XCTestCase {
   private enum Contexts {
-    static let all = ["empty", "all"]
+    static let all = ["empty", "array", "configuration", "all"]
```

Comment:
To avoid churn of output files in future changes, we can just test `empty` and `all`. Note: don't forget to delete the corresponding generated output files for `array` and `configuration`.